central/complianceoperator/v2/report/manager/complianceReportgenerator/report_gen_impl.go

```diff
@@ -84,19 +91,19 @@ type ResultEmail struct {
 
 func (rg *complianceReportGeneratorImpl) ProcessReportRequest(req *ComplianceReportRequest) {
 
-	data, err := rg.getDataForReport(req)
+	data, err := rg.getDataforReport(req)
 	if err != nil {
-		log.Errorf("Error getting report data for scan config %s", req.ScanConfigName)
+		log.Errorf("Error getting report data for scan config %s:%s", req.ScanConfigName, err)
 		return
 	}
 	if data == nil {
-		log.Errorf("Error getting report data for scan config %s", req.ScanConfigName)
+		log.Errorf("Error getting report data for scan config %s. NO data returned by query", req.ScanConfigName)
```

Comment:
This may be an issue OR it may be there is no data. 🤷‍♂️ 
```suggestion
		log.Infof("Unable to get report data for scan config %s. NO data returned by query", req.ScanConfigName)
```